View Controllers/ListViewController.m

```diff
@@ -12,6 +12,10 @@
 #import "HMSegmentedControl.h"
 #import "ProfileListCell.h"
 #import "DetailsViewController.h"
+#import "NCHelper.h"
+
+static NSString* NO_FAVORITE_MSG = @"You have no favorites!";
```

Comment:
it's standard practice to start these constants with a (which is weird, but whatever) 

for example, this one would be static const NSString *kNoFavoriteMsg

Reply:
Thanks a lot for the feedback!
For declaring string constants, I am getting a warning when assigning a label's text to the string constant, for example:
self.label.text = kNoFavoriteMsg;

The warning states that the label's text is expecting an NSString and not a constant NSString. I did some research and found two fixes:
- Cast kNoFavoriteMsg to NSString ( self.label.text = (NSString *)kNoFavoriteMsg; )
- Or declaring the constant this way: static NSString * const [...] instead of static const NSString * [...]

I just wanted to know which one of these would be better practice.